jwst/lib/set_telescope_pointing.py

```diff
@@ -85,13 +143,86 @@
                             'm_v2siaf',           # V to SIAF
                             'm_eci2siaf'          # ECI to SIAF
                         ])
-Transforms.__new__.__defaults__ = (None, None, None, None, None,
-                                   None, None, None, None)
+Transforms.__new__.__defaults__ = ((None,) * 10)
+
 # WCS reference container
 WCSRef = namedtuple('WCSRef', ['ra', 'dec', 'pa'])
 WCSRef.__new__.__defaults__ = (None, None, None)
 
 
+@dataclass
+class TransformParameters:
+    """Parameters required the calculations
+
+    Parameters
+    ----------
+    allow_default: bool
```

Comment:
Isn't there supposed to be whitespace before the colon in all of the parameter entries?

Reply:
✅